src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File from = new File(fromFileName);
+        File to = new File(toFileName);
+        String fileData = fileReader(from);
+        String[] rows = fileData.split(System.lineSeparator());
+        int supplyValue = 0;
+        int buyValue = 0;
+        for (String row : rows) {
+            String[] values = row.split(",");
+            if (values[0].equals("supply")) {
+                supplyValue += Integer.parseInt(values[1]);
+            } else if (values[0].equals("buy")) {
+                buyValue += Integer.parseInt(values[1]);
+            } else {
+                throw new RuntimeException("Wrong name value " + values[0]);
+            }
+        }
+        String report = "supply," + supplyValue + System.lineSeparator()
+                + "buy," + buyValue + System.lineSeparator()
+                + "result," + (supplyValue - buyValue);
+        fileWriter(to, report);
+    }
+
+    private String fileReader(File file) {
```

Comment:
```suggestion
    private String readFile(File file) {
```
для методі зазвичай використовуються дієсолова (іменники для класів)